sdk/synapse/azure-synapse-artifacts/swagger/README.md

```diff
@@ -2,42 +2,42 @@
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/pipelines.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/sparkJobDefinitions.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/sqlPools.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/sqlScripts.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/triggers.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/workspace.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/DataFlow.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Dataset.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/LinkedService.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Notebook.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Pipeline.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/SparkJobDefinition.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/SqlScript.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/0da14ec4f475435b9d7d0ba06e49f05a5daac226/specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Trigger.json
 
 payload-flattening-threshold: 1
 output-folder: ../
 package-name: azure-synapse-artifacts
 namespace: azure.synapse.artifacts
 license-header: MICROSOFT_MIT_NO_VERSION
-package-version: 0.19.0
+package-version: 0.20.0
```

Comment:
Please update dependencies of setup.py like https://github.com/Azure/azure-sdk-for-python/blob/23960894060e3017ec6233b630d2f3b10df9ea80/sdk/deviceregistry/azure-mgmt-deviceregistry/setup.py#L76-L81

Reply:
update setup.py file